src/DateTimePicker.elm

```diff
@@ -883,10 +974,33 @@ calendar pickerType state currentDate =
                             tr [] (List.map toCell week)
 
                         body =
-                            tbody [ class [ Days ] ]
-                                (List.map toWeekRow days)
+                            tbody [] (List.map toWeekRow days)
                     in
-                    table [ class [ Calendar ] ]
+                    Html.table
+                        [ css
+                            [ backgroundColor (hex "#ffffff")
+                            , Styles.tableStyle
+                            , width auto
+                            , margin (px 0)
+                            , descendants
+                                [ Css.Foreign.thead
```

Comment:
why not just add the styles to `thead` above?

Reply:
Yeah there are a bunch of places where we could refactor out the `Foreign` calls. I decided to err on the side of the MVP upgrade on this. 😅